packages/core/src/mm/actors/Player.S

```diff
@@ -1224,3 +1224,7817 @@ PATCH_END
 PATCH_START 0x8085e958
 .int 0
 PATCH_END
+
+/* Hammer Related ASM */
+
+PATCH_START 0x80123960
```

Comment:
I believe this patch can be removed and replaced with `PATCH_FUNC(0x80123960, Player_CustomActionToModelGroup)`

Reply:
Thank you for pointing this out. That patch was written before I found out that PATCH_FUNC existed :p